service/frontend/configs/quotas.go

```diff
@@ -95,13 +94,22 @@ var (
 
 	VisibilityAPIPrioritiesOrdered = []int{0}
 
+	// Special rate limiting for APIs that may insert replication tasks into a namespace replication queue.
+	// The replication queue is used to propagate critical failover messages and this mapping prevents flooding the
+	// queue and delaying failover.
+	NamespaceReplicationInducingAPIToPriority = map[string]int{
+		"UpdateNamespace":                  0,
```

Comment:
what about RegisterNamespace?

Reply:
We don't care about that one, it's called once per namespace so it can't flood the per namespace replication queue.

Reply:
?  there is no per-namespace queue, there's only one queue

Reply:
Yes, I was wrong.